lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -40,7 +39,7 @@
 
     <div class="cohort-management-supplemental">
         <p class="">
-            <i class="icon fa fa-info-circle"></i>
+            <i class="icon fa fa-info-circle" aria-hidden="true"></i>
```

Comment:
Since icons are visual and since there's text on this button that describes the action, we want to aurally hide this icon.